### PR TITLE
nano: update to 7.0

### PR DESCRIPTION
**Pull request by thecesrom**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.7 19H2026 x86_64
Command Line Tools 12.4.0.0.1.1610135815

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by thecesrom** [comment]
> [macos-11](https://github.com/macports/macports-ports/actions/runs/3474480026)
> GitHub Actions has encountered an internal error when running your job.

Please re-run failed jobs,

**Review by aeiouaeiouaeiouaeiouaeiouaeiou — CHANGES_REQUESTED** [review]

File: editors/nano/Portfile
Comment (thecesrom) [reply editors/nano/Portfile:7]: Restored `revision`.

Comment (thecesrom) [reply editors/nano/Portfile:7]: Please let me know if you'd like me to squash both commits.

Comment (aeiouaeiouaeiouaeiouaeiouaeiou) [reply editors/nano/Portfile:7]: Yes, that would be good.

Comment (thecesrom) [reply editors/nano/Portfile:7]: Done

Comment (aeiouaeiouaeiouaeiouaeiouaeiou) [reply editors/nano/Portfile:7]: @reneeotten

Comment (reneeotten) [reply editors/nano/Portfile:7]: ehm, why am I mentioned here? Even though the port is `openmaintainer` we do allow the usual 72h for the maintainer to take a look before merging (especially since this version jump would typically be considered a major update). So if you expected me to merge this PR that's not going to happen - you can try and ping again after the usual three days ;)

**Pull request merged**
